### ISSUE TITLE
Split client-only parts of book.json and move them to the resource pack

### ISSUE THREAD

**Issue by HyperDashPony** [VazkiiMods/Patchouli#701]
### Describe the feature

Adding the ability to change the font using a resourcepack rather than through data by modifying an original book would afford a lot more flexibility to modpack creators.

**Comment by williewillus** [comment]
Yes, we should split the clientside stuff of book.json out into a separate json file that resides in the resource pack. I'll rename this issue to point at what should be done.
